content/40.inequality.md

```diff
@@ -1,67 +1,125 @@
 This hypothesis is supported by some preliminary evidence showing that female patients who recovered from severe COVID-19 had higher antibody titers than males [@doi:10.1371/journal.ppat.1008570].
 Sex steroids can also bind to immune cell receptors to influence cytokine production [@doi:10.1186/s13293-020-00304-9].
-Additionally, social factors may influence risks related to both age and sex: for example, older adults are more likely to live in care facilities, which have been a source for a large number of outbreaks [@doi:10.1093/qjmed/hcaa136], and gender roles may also influence exposure and/or susceptibility due to differences in care-taking and/or risky behaviors (e.g., caring for elder relatives and smoking, respectively) [@doi:10.1186/s13293-020-00304-9] among men and women (however, it should be noted that both transgender men and women are suspected to be at heightened risk [@url:https://escholarship.org/uc/item/55t297mc].)
+Thus, both age and sex are thought to have some intrinsic relationship to risk, but also to be correlated with social factors contributing to risk, although it should be noted that both transgender men and women are suspected to be at heightened risk [@url:https://escholarship.org/uc/item/55t297mc].
 
 #### Comorbid Health Conditions
 
-A number of pre-existing or comorbid conditions have repeatedly been identified as risk factors for more severe COVID-19 outcomes.
+A number of pre-existing or comorbid conditions have repeatedly been identified as risk factors associated with more severe COVID-19 outcomes.
 Several underlying health conditions were identified at high prevalence among hospitalized patients, including obesity, diabetes, hypertension, lung disease, and cardiovascular disease [@doi:10.15585/mmwr.mm6915e3].
-Higher Sequential Organ Failure Assessment (SOFA) scores have been associated with a higher probability of mortality [@doi:10/ggnxb3], and comorbid conditions such as cardiovascular and lung disease as well as obesity were also associated with an increased risk of hospitalization and death, even when correcting for age and sex [@doi:10.1136/bmj.m1985].
-Diabetes may increase the risk of lengthy hospitalization [@doi:10.1016/j.cmet.2020.04.021] or of death [@doi:10.1016/j.cmet.2020.04.021;@doi:10.1007/s00592-020-01546-0].
-[@doi:10.1111/dom.14057] and [@doi:10.1152/ajpendo.00124.2020] discuss possible ways in which COVID-19 and diabetes may interact.
+Higher sequential organ failure assessment (SOFA) scores have been associated with a higher probability of mortality [@doi:10/ggnxb3], and comorbid conditions such as cardiovascular and lung diseases as well as obesity were also associated with an increased risk of hospitalization and death, even when correcting for age and sex [@doi:10.1136/bmj.m1985].
```

Comment:
One thought I've been having while reading through this is the use of the word "obesity." I've been trying to read more about the supposed connection between obesity and COVID-19 infection/outcomes (like this review- @doi.org/10.1111/obr.13128) but I don't see mention (except for one included paper) of controlling for other health conditions a person might have (type II diabetes, hypertension, cardiovascular disease) or their race, socioeconomic status, and access to healthcare or discrimination experienced while seeking healthcare. For we know the characterization of obesity as a disease is steeped in racism, and that people of higher weights experience excessive discrimination when seeking healthcare (for example- an H1N1 study found a correlation between higher weight and complications/ risk of death, but also that higher weight people were less likely to be prescribed early antiviral treatment...thus indicating healthcare discrimination could be a confounding variable @doi.org/10.1080/23744235.2016.1201721). I feel like obesity is often used as a proxy when researchers/clinicians are really talking about metabolic disturbances or other health conditions... just wondering if you all want to rethink the use of the word/ this framing in the paper. I'm happy to do more research on the topic or help rephrase things if you would like! I found these articles really helpful for learning more: url:https://sph.umich.edu/pursuit/2018posts/3-ways-public-health-professionals-can-combat-weight-stigma.html; url: https://www.huffpost.com/entry/fat-covid-19-pandemic-obesity_l_5f736f60c5b6e99dc3336e3e

Reply:
This is just my opinion, but I think that in the context of a review it would be better to continue to say "obesity."  We could of course add that the problem is probably metabolic conditions associated with the high weight, rather then the weight itself, but I don't think we should change what the original research papers reported.

Reply:
I would be okay with saying "high BMI" if you feel very strongly that the word "obesity" itself is pejorative, but I don't see how that would actually help.

Reply:
To maybe give some context to your observations @juliettemarie0405 

It is proposed that the underlying link between obesity and increased risk of COVID-19 is low-grade or systemic inflammation. Hence why other conditions such as diabetes, hypertension, CVD, and aging are also associated with increased risk of COVID-19 (all characterized by systemic inflammation-https://www.nature.com/articles/s41366-020-0597-4; and https://www.cdc.gov/coronavirus/2019-ncov/need-extra-precautions/people-with-medical-conditions.html). Many papers don't control for one or the other because many people have more than 1 of these conditions. You can be both type II diabetic and obese and even have hyperlipidemia, which in turn would increase one's risk of COVID-19. Heaped together, many people that are overweight or obese have metabolic syndrome, which is a clustering of these conditions. These non-communicable diseases (NCDs) or comorbidites are more prevalent in African Americans and you can be sure that social determinants of health is one of the reasons for that. 

There was a similar increased risk associated with NCDs for H1N1, but in contrast to COVID-19, obesity and asthma were not associated with increased risk [https://doi.org/10.1371/journal.pmed.1001053].

You are right that obesity of course has connotations attached, but I think from a scientific standpoint it is fair to use the term. Particularly since it is a key component of metabolic syndrome. However, I take your point about from the example of the H1N1, there is certainly medical prejudice. Maybe we can acknowledge this somehow in the text.

Reply:
I think that some of the controversies that become associated with the idea of "risk factors" are related to the fact that different people and fields use the term in different ways.  Some use the term only when they believe there is a direct cause-and-effect relationship, while others use the 
term for anything that is a significant statistical predictor of an outcome (even if it might be a proxy for something else that's harder to measure).  It's unfortunate that the same word is used in those two different ways, but we cannot fix it here.  The former definition is more natural for what we ordinarily mean by risk, in the sense of somebody doing something dangerous.  But I think the latter definition is actually more common in the field.

Reply:
I think it would be great to link this back to the medical discrimination that @tlukan wrote about in text below -- and also, I know @RLordan and I are really hoping we can make it clear how inflammation (which we want to make sure is contextualized alongside experiences/environment because they are so closely tied together) ties a lot of fragments of the COVID-19 story together. In this case, obesity/high BMI/whatever we want to call it can be view as a signal of inflammation, but it is also associated (causally or not) with other factors that can influence health outcomes (medical discrimination, as @juliettemarie0405 mentioned, or also racial discrimination since BMI is not an inclusive measure, also often prevalent in places with limited access to health-related resources and/or low SES, etc.)

One thing I was hoping to do while I was writing this draft was to try to drag any "I'm a biologist, I don't care about social sciences" people reading it along with us by working out in circles from things they would view as biological (e.g., obesity) and eventually tying it in to systemic effects that elevate inflammation. It seems like I need to revisit this because it's not working the way it is! I really appreciate all the feedback (and the references @juliettemarie0405!)

Reply:
All true, I think it would be fair to say in my world we often dip into both definitions blindly haha

Reply:
Also, welcome @juliettemarie0405 !

Reply:
Thank you @RLordan! And thank you for that very helpful explanation of the role of inflammation in all of this, and for pointing out the difficulties in controlling for comorbidities. I really appreciate the opportunity to discuss this with all of you, and I am excited by the idea of introducing this discussion to readers as well. @rando2 I really like how you framed it- that the underlying thread in all of this is inflammation, that there are many causative forces, and that higher weight should be reconceptualized as a "signal of inflammation" rather than a condition in and of itself. Even if you all decide to move forward with the term obesity, I really like @RLordan's suggestion of acknowledging how medical prejudice (and racism, poverty, fatphobia) inherently complicate how we understand associations identified in the literature.